Makefile

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
```

Comment:
I think you should not enable Travis for your repository: https://github.com/ruflin/gotestcover .
Github/Travis will run a new test on my repository when you submit a pull request.
Of course, it's not currently enabled...